### DISCUSSION TITLE
Local Install With Local Web Interface

### DISCUSSION THREAD

**Discussion by Jimily412** [gameserverapp/Platform#892] [Suggestions]
I understand this is a long shot and kind of defeats some of the purpose of what you've got going here. But I think a little redundancy could be added to the app by having some kind of local installation, where you pull a git repository and install the app and create a sort of web server that is accessed by typing the local IP address into the browser. I know it sounds like I have no idea what I am saying, I probably don't. To give some reference, something similar to plex or tautulli or unmanic where the dashboard is accessed through a local IP. Like I said, this is a long shot and definitely not something that would happen quickly. But I believe there could be some discussion to be had on this topic.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion! 

GSA is a SaaS service, so we are a hosted service by design. There are no plans for locally hosted dashboards. There are open source projects that do support this, if you are looking for a local setup.
